backend-server/src/api/versions/v1.ts

```diff
@@ -6,6 +6,6 @@ import { interviewerRouter } from "../routes/interviewer";
 
 export const v1Router = express.Router();
 
-v1Router.use("/interviewer", interviewerRouter);
-v1Router.use("/availability", availabilityRouter);
-v1Router.use("/event", eventRouter);
+v1Router.use("/interviewers", interviewerRouter);
```

Comment:
Why did we change this? Can we keep them all singular or change them all to plural for consistency? Also newline at the end

Reply:
apparently plural endpoints are more restful, i'll update them to all be plural though